### ISSUE TITLE
Samples for Unreal Engine? 

### ISSUE THREAD

**Issue by ElizzaRg** [GPUOpen-Effects/FEMFX#7]
Where can I find samples scene of code usage in the Unreal ENgine engine?

**Comment by stoyky** [comment]
I would love to play around with this as well

**Comment by spammaleros** [comment]
Same question

**Comment by elarsen-amd** [comment]
I just added information to the README about accessing the Unreal sample

**Comment by ElizzaRg** [comment]
> I just added information to the README about accessing the Unreal sample

Thanks a lot!
Tell me please, where can I find an example of using a plug-in with machine deformation?
This is a very exciting demo that haunts me. Also a demo with the destruction of the tree!

**Comment by stoyky** [comment]
Thanks for adding links to the AlienPods project in UE4. However I think everyone here, myself included, is referring to the sample projects that were shown in the talk, such as for instance the melting rubber ducks, the deformable car etc. Is there any chance to release these as well?

**Comment by DemionBlack** [comment]
Hey I made a direct request to AMD Munich, we are considering implementing FEMFX in our game, but our chief developer only found a version for UE 4.18 - can you also get a version for 4.24?

**Comment by noodles0026** [comment]
Is the UE4 FEMFX plugin able to interact with the rigid bodies within the Editor?

I did manage to recreate the scene setup in Alien Pods Project sample and upon investigating, the player first person pawn only interacts with the UE4 rigid bodies and the FEMFX objects are on their own collisions with FEM objects. I was wondering if the interfacing said in the presentation will be able to interact with terrains as those are vital in large scale scenes, unless the intended workflow is primarily FEM exclusive or on a smaller scale. But if that is the case then it seems too redundant to make a separate collision objects with FEM settings on top of existing terrains just for FEM interactions and collisions. But then again, it might not be the case on future versions of the plugin. 

There is a scene object called FEMRigidBodyScene and I am wondering if this is where the settings for the rigid body interaction but upon deleting it, the scene still works fine.

**Comment by ElizzaRg** [comment]
Hi! Please demo scene for Unreal Engine 4 -> Denting metal

**Comment by Andrey-Pavlov** [comment]
I'm curious why it's always a problem to support the latest UE4 version for physics library and engines? The same happened with Nvidia FLEX, just demo for the specific UE4 version and nothing more. How to develop games if FLEX and FEMFX is always "outdated" for UE?

**Comment by Khyretos** [comment]
> I just added information to the README about accessing the Unreal sample
> 
> 

Yes but is there a way to make assets without Houdini like blender or Maya?

**Comment by Khyretos** [comment]
> Thanks for adding links to the AlienPods project in UE4. However I think everyone here, myself included, is referring to the sample projects that were shown in the talk, such as for instance the melting rubber ducks, the deformable car etc. Is there any chance to release these as well?

hey if you want the demo showed in the video, you can download it here, it is the button at the bottom:
https://www.amd.com/en/technologies/tressfx-femfx-agreement

but unfortunatly it is not the source code it is the demo itself, they apparently do not want to provide you with anything usefull, alienpod itself is a cool demo but when trying to do anything else like adding a .FEM mesh or anything it just crashes.

but at least its fun to play around with

**Comment by epicdud3** [comment]
> > Thanks for adding links to the AlienPods project in UE4. However I think everyone here, myself included, is referring to the sample projects that were shown in the talk, such as for instance the melting rubber ducks, the deformable car etc. Is there any chance to release these as well?
> 
> hey if you want the demo showed in the video, you can download it here, it is the button at the bottom: https://www.amd.com/en/technologies/tressfx-femfx-agreement
> 
> but unfortunatly it is not the source code it is the demo itself, they apparently do not want to provide you with anything usefull, alienpod itself is a cool demo but when trying to do anything else like adding a .FEM mesh or anything it just crashes.
> 
> but at least its fun to play around with

of course the demo is gone now, do you have something like a personal archive of the demo or something? or is it just lost media lol

**Comment by Khyretos** [comment]
> > > Thanks for adding links to the AlienPods project in UE4. However I think everyone here, myself included, is referring to the sample projects that were shown in the talk, such as for instance the melting rubber ducks, the deformable car etc. Is there any chance to release these as well?
> > 
> > 
> > hey if you want the demo showed in the video, you can download it here, it is the button at the bottom: https://www.amd.com/en/technologies/tressfx-femfx-agreement
> > but unfortunatly it is not the source code it is the demo itself, they apparently do not want to provide you with anything usefull, alienpod itself is a cool demo but when trying to do anything else like adding a .FEM mesh or anything it just crashes.
> > but at least its fun to play around with
> 
> of course the demo is gone now, do you have something like a personal archive of the demo or something? or is it just lost media lol

Sorry i deleted it long time ago since i went over to Godot.

**Comment by epicdud3** [comment]
> > > > Thanks for adding links to the AlienPods project in UE4. However I think everyone here, myself included, is referring to the sample projects that were shown in the talk, such as for instance the melting rubber ducks, the deformable car etc. Is there any chance to release these as well?
> > > 
> > > 
> > > hey if you want the demo showed in the video, you can download it here, it is the button at the bottom: https://www.amd.com/en/technologies/tressfx-femfx-agreement
> > > but unfortunatly it is not the source code it is the demo itself, they apparently do not want to provide you with anything usefull, alienpod itself is a cool demo but when trying to do anything else like adding a .FEM mesh or anything it just crashes.
> > > but at least its fun to play around with
> > 
> > 
> > of course the demo is gone now, do you have something like a personal archive of the demo or something? or is it just lost media lol
> 
> Sorry i deleted it long time ago since i went over to Godot.

is there anywhere i can find the demo to download it or is it actually lost media now?

**Comment by epicdud3** [comment]
i guess not, damn.
